src/test/java/uk/gov/dhsc/htbhf/steps/ScotlandSteps.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.dhsc.htbhf.steps;
+
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import uk.gov.dhsc.htbhf.page.ScotlandPage;
+
+/**
+ * Steps for the do you live in Scotland page
+ */
+public class ScotlandSteps extends CommonSteps {
+
+    @When("^I say No to the do you live in Scotland question")
+    public void selectNoToDoYouHaveChildren() {
```

Comment:
Copy and paste error, should be selectNoToDoYouLiveInScotland(). Same for method below.

Reply:
Have corrected, thanks.